tool/lib.js

```diff
@@ -11,9 +12,11 @@ exports.parsePlist = function(xmlOrJSON, callback) {
         plist.parseString(xmlOrJSON, function(_, result) {
             json = result[0];
         });
-    } else {
+    } else try {
```

Comment:
That indentation looks a bit too irregular to me. It kinda works but to the layman might even suggest something like the catch applies to the whole if/else or something. It just looks a bit odd. I would just do brackets in the `else`.